src/store/modules/management.js

```diff
@@ -92,8 +93,16 @@ const mutations = {
     state.managedSubUnits = subUnitsUpdatedNotification.subUnitsList.map(
       s => new SubUnit(s.id, s.name)
     );
+
     if (state.selectedSubUnit === null) {
       state.selectedSubUnit = state.managedSubUnits[0];
+    } else {
+      let unitsWithId = state.managedSubUnits.filter(s => s.id === state.selectedSubUnit.id);
+      if (unitsWithId.length > 0) {
+        state.selectedSubUnit = unitsWithId[0];
+      } else {
+        state.selectedSubUnit = state.managedSubUnits[0];
+      }
     }
   },
   [M.SELECT_MANAGED_FUNCTION]({ commit }, managedFunction) {
```

Comment:
the mutator parameter "managedFunction" is never used. perhaps it should be removed

Reply:
but it is used :(